src/AutoRest.CSharp/Common/Output/Models/Types/ModelFactoryTypeProvider.cs

```diff
@@ -177,6 +187,13 @@ private Method CreateMethod(SerializableObjectType model)
                     continue;
                 }
 
+                if (ctorParameter.IsRawData)
+                {
+                    // we do not want to include the raw data as a parameter of the model factory entry method, therefore here we skip the parameter, and use empty dictionary as argument
```

Comment:
we are putting `null` not empty dictionary here, right?

Reply:
Yes we did this intentionally.
I discussed this with Michael. Kind of it is an internal only property, customer will not have the chance to touch it, therefore if we could do the null check carefully, it is OK that we assign null.

Reply:
What's the benefit using null instead of empty dictionary here? we don't need to check null afterwards during serialization generation.

Reply:
> What's the benefit using null instead of empty dictionary here? we don't need to check null afterwards during serialization generation if empty dictionary is always in place.

less allocation